oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/entity/LanguageTrans.java

```diff
@@ -33,6 +33,10 @@ public Language value(String language) {
                 return Language.PYTHON;
             case "ruby":
                 return Language.RUBY;
+            case "go":
+                return Language.GO;
+            case "Lua":
```

Comment:
I didn't notice this should be lower case. How about to use `toLowerCase` in the switch?

Reply:
done.